datadog_checks_base/datadog_checks/base/checks/win/wmi/__init__.py

```diff
@@ -129,6 +147,10 @@ def _get_tag_query_tag(self, sampler, wmi_obj, tag_query):
         self.log.debug(u"Extracted `tag_queries` tag: '%s'", tag)
         return tag
 
+    def extract_metrics(self, constant_tags):
```

Comment:
Used by wmi_check

Reply:
Seems like it's not actually used there yet (`WMICheck` calls into `_extract_metrics`). Are you planning to use this method there afterwards, or is it an oversight?

Reply:
Yes, the plan was to call extract_metrics instead but I'll define it there instead